src/data/device_automation.ts

```diff
@@ -0,0 +1,39 @@
+import { HomeAssistant } from "../types";
+
+export interface DeviceTrigger {
+  platform: string;
+  device_id: string;
+  domain: string;
+  entity_id: string;
+  type?: string;
+  event?: string;
+}
+
+export interface DeviceTriggerList {
```

Comment:
Why is the return value an object with a list, why not just directly a list ?

Reply:
Hmm, fixed.